content/providers/provider-azure/authentication.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Authentication 
+weight: 10
+description: Authentication options with the Upbound Azure official provider
+---
+
+The Upbound Official Azure Provider supports multiple authentication methods.
+
+* [Service principal with Kubernetes secret]()
+* [System-assigned managed identity]()
+* [User-assigned managed identity]()
+
+## Service principal with Kubernetes secret
+
+A service principal is an application within the Azure Active Directory that
+passes `client_id`, `client_secret`, and `tenant_id` authentication
+tokens to create and manage Azure resources. 
+
+### Create a service principal using the Azure CLI tool
+
+First, find the Subscription ID for your Azure account.
+
+```shell
+az account list
+```
+
+Note the value of the `id` in the return output. 
+
+Next, create a service principle `Owner` role. Update the `<subscription_id>`
+with the `id` from the previous command.
+
+```shell
+az ad sp create-for-rbac --sdk-auth --role Owner --scopes /subscriptions/<subscription_id> \
+  > azure.json
+```
+
+The `azure.json` file in the preceding command contains the client ID, secret, and
+tenant ID of your subscription.
+
+
+Next, use `kubectl` to associate your Azure credentials file with a generic
+Kubernetes secret.
+
+```shell
+kubectl create secret generic azure-secret -n upbound-system --from-file=creds=./azure.json
+```
+
+## Configure your provider
+
+Apply these changes to your `ProviderConfig` file. 
+
+```yml
+apiVersion: azure.upbound.io/v1beta1
+metadata:
+  name: default
+kind: ProviderConfig
+spec:
+  credentials:
+    source: Secret
+    secretRef:
+      namespace: upbound-system
+      name: azure-secret
+      key: creds
+```
+
+Your credential `source` must be `Secret` and you must specify the namespace,
+name, and key if you used different values.
+
+Apply your configuration.
+
+```shell
+kubectl apply -f
+```
```

Comment:
I'd omit this and assume they know how to apply a kube setting, mainly to avoid arguments about which command, the file name, blah blah blah.